index.html

```diff
@@ -136,6 +136,164 @@ <h2>
+        </li>
+        <li>A <dfn>maximum analog value</dfn> representing full activation. The
+        largest in-bounds [=analog button value=].
+        </li>
+      </ul>
+      <p>
+        A [=button=] may be capable of detecting touch. If so, the [=button=]
+        has a <dfn>button touch value</dfn> [=input value=] that is `true` when
+        the [=button=] is touched and `false` otherwise.
+      </p>
+      <p>
+        A [=gamepad=] may have [=touch surfaces=]. A <dfn>touch surface</dfn>
+        is an input control that provides 2D position data representing points
+        of contact. A [=gamepad=] has a <dfn>touch surface list</dfn> which is
+        a [=list=] containing the [=touch surfaces=] for the [=gamepad=]. The
+        list is ordered such that [=touch surfaces=] closer to the left side of
+        the [=gamepad=] appear closer to the start of the list.
+      </p>
+      <p>
+        A [=touch surface=] has an <dfn>active touch point list</dfn> [=input
+        value=], a [=list=] of zero or more [=touch points=] representing the
+        points of contact currently detected by the sensor, ordered such that
+        earlier [=touch points=] appear closer to the start of the [=list=]. A
+        <dfn>touch point</dfn> represents a single point of contact at a single
+        point in time. A [=touch point=] has a <dfn>touch x coordinate</dfn>
+        representing its position along the user's left-right axis and a
+        <dfn>touch y coordinate</dfn> representing its position along the
+        perpendicular axis.
+      </p>
+      <p>
+        A [=touch surface=] may have surface dimension [=input values=]. The
+        <dfn>surface width</dfn> and <dfn>surface height</dfn> [=input values=]
+        are the dimensions of the [=touch surface=] in the same units as the
+        [=touch x coordinate=] and [=touch y coordinate=]. A [=touch surface=]
+        has both dimension values or neither value.
+      </p>
+      <p>
+        A [=touch point=] may be a new contact point or a continuation of an
+        earlier contact. A [=touch point=] is <dfn>part of an existing active
+        touch point</dfn> if the [=user agent=] identifies that it is a
+        continuation of a [=touch point=] represented by an earlier
+        {{GamepadTouch}}. The <dfn>active touch point id</dfn> for a [=touch
+        point=] is the {{GamepadTouch/touchId}} of the earlier
+        {{GamepadTouch}}.
+      </p>
+      <p>
+        A [=gamepad=] may have [=haptic actuators=]. A <dfn>haptic
+        actuator</dfn> is an output control capable of moving the [=gamepad=]
+        in a way that can be felt by the user. A <dfn>haptic effect</dfn> is
+        the use of one or more [=haptic actuators=] to provide feedback to the
+        user. The [=user agent=] is responsible for commanding [=haptic
+        actuators=] to play and stop [=haptic effects=] on [=available=]
+        [=gamepads=].
```

Comment:
Maybe this way?
```suggestion
        A [=gamepad=] may have [=haptic actuators=]. A <dfn>haptic
        actuator</dfn> is an output control capable of moving the [=gamepad=]
        in a way that can be felt by the user. One or more [=Haptic actuators=]
        can be used to generate <dfn>haptic effects</dfn> and provide feedback to the
        user. The [=user agent=] is responsible for commanding [=haptic
        actuators=] to play and stop [=haptic effects=] on [=available=]
        [=gamepads=].
```

Reply:
Yes, "the use of" was awkward and this is more fluid. Modified to emphasize that a single effect can be composed of vibrations from multiple actuators:

> A [=haptic actuator=] can be used to generate a **haptic effect** that provides feedback to the user. Vibrations from multiple actuators are combined to generate more complex haptic effects.